pilot/pkg/serviceregistry/kube/controller/endpointcontroller.go

```diff
@@ -42,16 +46,19 @@ type kubeEndpointsController interface {
 
 // kubeEndpoints abstracts the common behavior across endpoint and endpoint slices.
 type kubeEndpoints struct {
-	c        *Controller
-	informer cache.SharedIndexInformer
+	c          *Controller
+	informer   cache.SharedIndexInformer
+	needResync map[string]sets.Set
 }
 
 func (e *kubeEndpoints) HasSynced() bool {
 	return e.informer.HasSynced()
 }
 
 func (e *kubeEndpoints) Run(stopCh <-chan struct{}) {
+	fmt.Println("Run ^^^^^")
```

Comment:
Is this intentional or just for debugging?

Reply:
Sorry. Good catch. Leftover from my debugging. Removed now